src/monitoring/bin/db_full_check.sh

```diff
@@ -0,0 +1,8 @@
+echo CROMWELL_GOOGLE_PROJECT is ${CROMWELL_GOOGLE_PROJECT}
+echo CROMWELL_CLOUDSQL_INSTANCE is ${CROMWELL_CLOUDSQL_INSTANCE}
+echo CROMWELL_MAXIMUM_DATABASE_SIZE_TIB is ${CROMWELL_MAXIMUM_DATABASE_SIZE_TIB}
+echo CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS is ${CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS}
+# echo CROMWELL_METRICS_ACCESS_TOKEN is ${CROMWELL_METRICS_ACCESS_TOKEN}
```

Comment:
Probably best to remove this altogether? Or if this is a sanity check, maybe do this as a "does the value exist" check rather than an echo?